pkg/operatorstatus/operatorstatus.go

```diff
@@ -89,9 +108,11 @@ func (o *ClusterOperatorHandler) UpdateOperatorStatus(cfg *v1.Config) error {
 		"",
 		reasonForProgressing)
 	if err != nil {
-		return nil
+		// note error but try the other conditions
+		returnError = err
+		logrus.Warningf("error occurred while attempting to set progressing condition: %s", err.Error())
 	}
-	return nil
+	return returnError
```

Comment:
Might be nice to roll all of the errors into a single error.

Reply:
part of next push